docs/PROXY_WASM.md

```diff
@@ -681,9 +681,8 @@ limitations on some supported features:
     - `on_http_request_body`
     - `on_http_call_response`
 
-2. The key/value and queue shared memory implementations do not implement an LRU
-   eviction mechanism when the allocated memory slab is full:
-    - `proxy_set_shared_data`
+2. The "queue" shared memory implementation does not implement an LRU eviction
```

Comment:
```suggestion
2. The "queue" shared memory implementation does not implement an automatic eviction
```

A bit pedantic, but I don't think LRU would make sense for a queue, after all you "use" by popping; if one were to evict rather than reject a push, the only two options I see would be to drop either the front (remove the youngest element of the queue, i.e., discard the pushed element silently) or the back (remove the oldest element of the queue — IMO it would be a stretch to call that "LRU").

Reply:
Yes I changed this a bit too fast; was planning on reviewing this morning but you hashed it out!